aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -302,7 +285,7 @@ export const ShippingExample = () => {
                 mask={phoneMask}
               />
             </FormField>
-            <RequiredFormField
+            <FormField
```

Comment:
I am having a hard time tagging line 212 which reads `Shipping Address *`.

I am noticing two things:
- One, the visual treatment of the label and * is different than the new execution
- Two, how should we approach this in general? We should probably fix it while we are here.

```
            <FormField required htmlFor="firstName" name="firstName" label="Shipping Address">
              <TextInput
                id="firstName"
                name="firstName"
                placeholder="First Name"
              />
            </FormField>
```

and remove
```
<Text
              margin={{ horizontal: 'none', vertical: 'xsmall' }}
              size="xsmall"
            >
              Shipping Address *
            </Text>
```

Reply:
This example is very wrong for accessibility. Glad you called it out. Ideally here, we'd want to be using a `fieldset` and group the inputs together where their individual labels are hidden (but still accessible by screen reader). Grommet doesn't currently support `fieldset` or visually-hidden labels. This example requires a bit more work, and I wonder if it would be better to remove it from the page for now until we can work on an improvement to the example?

Reply:
Yes. We seem to keep kicking this can down the road. We should raise it as an issue / create an issue for it so that we make sure it gets addressed.

I'd be okay removing temporarily, but would want us to notify the team for the decision + the rationale + ask for any objections. And, also create an issue to talk through / propose a fieldset solution in Grommet. --> which could be something like:
```
const FormFieldSet = ( {children} ) => {
  return <fieldset ...>
    <legend ... />
    {children}
  </fieldset>;
};
```
I don't know all the fieldset options off the top of my head, plus whatever styling would be necessary.

Reply:
Thinking more about the fieldset implementation above, I think we'd still need the hidden labels functionality though, right?

Reply:
Yep we'd still need hidden labels as well. I'll file an issue if we don't have one already for fieldset work and raise it as an issue in planning next week. I'll leave the example in the page for now.